docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,296 @@
+
+Ownership Strength Policy is mutable, but Ownership Policy is not. Then, it is needed to configure EXCLUSIVE_OWNERSHIP_POLICY to all participants before running the ROS nodes.
+To do that, create a new xml file in the root of the workspace:
+
+.. code-block:: bash
+
+    cd ~/vulcanexus_ws
+    touch profiles1.xml
+
+Open the newly created file with your preferred editor and paste the following xml code:
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/profiles1.xml
+    :language: xml
+
+
+This xml includes one profile for a publisher (data writer) and one profile for a subscriber (data reader), and sets them to exclusive ownership, and ownership strength of value 10 for the publisher.
+This will be applied to the Publisher 1 and to the Subscriber.
+We need another profile in a separate file to assign a different ownership strength to the Publisher 2:
+
+.. code-block:: bash
+
+    touch profiles2.xml
+
+
+.. literalinclude:: /resources/tutorials/core/qos/mutable/profiles2.xml
+    :language: xml
+
+
+This will assign an ownership strength of value 2 to the Publisher 2.
+
+Build
+-----
+
+Now the package is ready to be built.
+Change your directory to the workspace folder and build using colcon:
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    cd ~/vulcanexus_ws
+    colcon build
+
+
+Run
+---
+
+Open three terminals in the workspace folder.
+On each, you need to source Vulcanexus installation, as well as the package installation.
+Then, export the `FASTRTPS_DEFAULT_PROFILES_FILE` environment variable to point out to the corresponding profiles file.
+And run the ROS 2 node.
+
+First, in the first terminal, run the subscriber node, configured with the profiles1.xml file:
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    cd ~/vulcanexus_ws
+    . install/setup.bash
+    export FASTRTPS_DEFAULT_PROFILES_FILE=./profiles1.xml
+    ros2 run vulcanexus_change_mutable_qos change_mutable_qos_sub
+
+
+Then, in another terminal, run the first publisher, configured also with the profiles1.xml file.
+This Publisher will then be configured with ownership strength value of 10.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    cd ~/vulcanexus_ws
+    . install/setup.bash
+    export FASTRTPS_DEFAULT_PROFILES_FILE=./profiles1.xml
+    ros2 run vulcanexus_change_mutable_qos change_mutable_qos_pub1
+
+
+At this point you will be able to see that both nodes are communicating, and the messages from Publisher 1 can be seen in the Subscriber.
+
+In the third terminal, run the second publisher, configured with the profiles2.xml file.
+This Publisher will then be configured with ownership strength value of 2.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/humble/setup.bash
+    cd ~/vulcanexus_ws
+    . install/setup.bash
+    export FASTRTPS_DEFAULT_PROFILES_FILE=./profiles2.xml
+    ros2 run vulcanexus_change_mutable_qos change_mutable_qos_pub2
```

Comment:
I still think these may be tabs if we put the needed explanations in a bash comment

Reply:
Done